inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -2342,3 +2298,81 @@ void MKLDNNGraphOptimizer::MergePermuteAndReorder(MKLDNNGraph &graph) {
         }
     }
 }
+
+static bool isReorderAvailable(const MKLDNNEdgePtr parentEdge, const MKLDNNEdgePtr childEdge, const mkldnn::engine& eng) {
+    MKLDNNMemoryDesc dstMemDesc(childEdge->getDesc());
+    MKLDNNMemoryDesc srcMemDesc(parentEdge->getDesc());
+
+    memory::primitive_desc dstPrimitiveDesc(dstMemDesc, eng);
+    memory::primitive_desc srcPrimitiveDesc(srcMemDesc, eng);
+    mkldnn::primitive_attr attr;
+
+    mkldnn_primitive_desc_t result = nullptr;
+    auto status = mkldnn_reorder_primitive_desc_create_v2(&result, srcPrimitiveDesc.get(), dstPrimitiveDesc.get(), attr.get());
+    if (result) {
+        mkldnn_primitive_desc_destroy(result);
+    }
+
+    return mkldnn_success == status;
+}
+
+void MKLDNNGraphOptimizer::AddConvertToReorder(MKLDNNGraph &graph) {
+    const auto& vecNodes = graph.GetNodes();
+    std::set<MKLDNNNodePtr> insertedReorders;
+
+    for (int i = 0; i < vecNodes.size(); i++) {
+        auto node = vecNodes[i];
+        if (Reorder == node->getType()) {
+            auto inpEdge = node->getParentEdgeAt(0);
+            auto outEdge = node->getChildEdgeAt(0);
+            // Some guard checks
```

Comment:
I would prefer to have more usefull comment or to remove it at all.

Reply:
I agree. But this chunk of code has been completely removed (see b995de5).